docs/docs/configuration/connecting-to-data-stores/lightstep.md

```diff
@@ -0,0 +1,98 @@
+# Lightstep
+
+If you want to use [Lightstep](https://lightstep.com/) as the trace data store, you'll configure the OpenTelemetry Collector to receive traces from your system and then send them to both Tracetest and Lightstep. And, you don't have to change your existing pipelines to do so.
+
+:::note
+It is important to notice that this relies on the [tailsampling](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor) processor, which, at the moment, is only available in the [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/) version of the collector.
```

Comment:
I think we removed the `tailsampling` support https://github.com/kubeshop/tracetest/pull/1806 Do we want to remove this note?